### PR TITLE
javascript-operators-and-expressions exercise - Khanh Nguyen

### PR DESCRIPTION
**Pull request by khanhnguyenq**
Screenshot 2023-10-23 at 2 26 46 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const width = 10;
+const height = 10;
+const area = width * height;
+
+console.log('value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+const bill = 100;
+const payment = 67;
+const change = bill - payment;
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js]: Your `payment` should be higher than the `bill`, and your `change` operation will need to be swapped

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Khanh!

**Review by shawnkost — APPROVED** [review]
👍🏻

File: javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (khanhnguyenq) [reply javascript-operators-and-expressions/main.js]: Thanks Shawn! See screenshots below!
Screenshot 2023-10-23 at 2 53 45 PM
Screenshot 2023-10-23 at 2 54 30 PM

**Pull request merged**